Sources/APPX.cpp

```diff
@@ -102,6 +104,15 @@ namespace appx {
             for (const auto &fileNamePair : fileNames) {
                 const std::string &archiveName = fileNamePair.first;
                 const std::string &fileName = fileNamePair.second;
+
+                const std::string suffix = "AppxBundleManifest.xml";
+                if (bundle &&
+                        suffix.size() < archiveName.size() &&
+                        std::equal(suffix.rbegin(), suffix.rend(), archiveName.rbegin())) {
```

Comment:
> the python script (windows_package_app.py) provides a pair of files for AppxManifest.xml and also StoreManifest.xml and they are provided in the same way all other files are provided.

Exactly. For the .appx file format, it's just another file.

> If we are going to change the way we treat these manifests, we would need to change it when we create both appx and appxbundle.

AppxBundleManifest.xml is the only special manifest. AppxManifest.xml isn't special for the .appx or .appxbundle formats.

All the special files (to my knowledge):

* [Content_Types].xml
* AppxBlockMap.xml
* AppxSignature.p7x
* AppxMetadata/CodeIntegrity.cat (not yet generated by fb-util-for-appx)
* AppxBundleManifest.xml (.appxbundle)
* *.appx (inspected when generating AppxBundleManifest.xml, excluded from AppxBlockMap.xml (for .appxbundle only?))

> Maybe we can take this offline.

Sure. I think there's something I'm missing. =S